specifications/api/swagger.yaml

```diff
@@ -2160,7 +2160,7 @@ x-tagGroups:
   - name: Shelley (Sequential)
     tags:
     - Wallets
-    - Addresses
+    - Shelley Addresses
```

Comment:
Erf, this sort of breaks the pattern :sweat_smile: ... Shall we maybe have a separate group under Misc called `Tools` or `Utils` ?